zebra/zebra_rnh.c

```diff
@@ -788,9 +951,8 @@ void zebra_print_rnh_table(vrf_id_t vrfid, afi_t afi, struct vty *vty,
  * free_state - free up the re structure associated with the rnh.
  */
 static void free_state(vrf_id_t vrf_id, struct route_entry *re,
-		       struct route_node *rn)
+		       struct route_node *rn, struct prefix *re_prefix)
```

Comment:
new arg 're_prefix' isn't used here?

Reply:
dunno anymore it's been like 2 months since I made these changes.  I'll have to rewrap my head around this.

Reply:
yeah this was a  code path that I decided to go another direction on and just didn't clean this up.  Removed.